spec/unit/cli_spec.rb

```diff
@@ -523,6 +502,58 @@
     end
   end
 
+  describe ".prepare_filter" do
+    context "with the --skip-files option" do
+      before(:each) do
+        allow(Filter).to receive(:from_default_definition).and_return(Filter.new)
+      end
+
+      it "it reads a list of excluded files from a file" do
+        FileUtils.mkdir_p("/tmp")
+        exclude_file = Tempfile.new('exclude_file')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.